<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sploty Obrazów</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_Picture">
       <property name="minimumSize">
        <size>
         <width>512</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Picture</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_FFT">
       <property name="minimumSize">
        <size>
         <width>512</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>FFT</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_openPicture">
        <property name="text">
         <string>Załaduj zdjęcie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_doFFT">
        <property name="text">
         <string>Wykonaj Splot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_swap">
        <property name="text">
         <string>Ustaw zdjęcie wyjściowe na wejście</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_maskSection">
        <property name="text">
         <string>Maska:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_predefinieMasks"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLineEdit" name="lineEdit_b1">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEdit_b3">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEdit_a3">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_c2">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineEdit_c3">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_a2">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="lineEdit_c1">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_b2">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_a1">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1234</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_openPicture</tabstop>
  <tabstop>pushButton_doFFT</tabstop>
  <tabstop>pushButton_swap</tabstop>
  <tabstop>comboBox_predefinieMasks</tabstop>
  <tabstop>lineEdit_a1</tabstop>
  <tabstop>lineEdit_a2</tabstop>
  <tabstop>lineEdit_a3</tabstop>
  <tabstop>lineEdit_b1</tabstop>
  <tabstop>lineEdit_b2</tabstop>
  <tabstop>lineEdit_b3</tabstop>
  <tabstop>lineEdit_c1</tabstop>
  <tabstop>lineEdit_c2</tabstop>
  <tabstop>lineEdit_c3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
